opl/get_kafka_times.py

```diff
@@ -78,26 +80,43 @@ def kafka_ts2dt(self, timestamp):
 
     def create_consumer(self):
         # Store Kafka config to status data
-        self.status_data.set("parameters.kafka.bootstrap", self.kafka_hosts[0])
+        self.status_data.set("parameters.kafka.bootstrap", self.kafka_host)
         self.status_data.set("parameters.kafka.group", self.kafka_group)
         self.status_data.set("parameters.kafka.topic", self.kafka_topic)
         self.status_data.set("parameters.kafka.timeout", self.kafka_timeout)
 
-        # Create Kafka consumer
-        consumer = KafkaConsumer(
-            self.kafka_topic,
-            bootstrap_servers=self.kafka_hosts,
-            auto_offset_reset="earliest",
-            enable_auto_commit=True,
-            group_id=self.kafka_group,
-            max_poll_records=self.kafka_max_poll_records,
-            session_timeout_ms=50000,
-            heartbeat_interval_ms=10000,
-            consumer_timeout_ms=self.kafka_timeout,
-        )
-        logging.debug(
-            f"Created Kafka consumer for {self.kafka_hosts} for {self.kafka_topic} topic in group {self.kafka_group} with {self.kafka_timeout} ms timeout"
-        )
+        try:
+            logging.info(
+                f"Creating consumer with sasl username&pasword to {self.kafka_host}"
+            )
+            consumer = KafkaConsumer(
+                self.kafka_topic,
+                bootstrap_servers=self.kafka_host,  # self.kafka_hosts,
```

Comment:
I think `bootstrap_servers` needs to be a list, not just one string.

Reply:
It works even as a single string: https://kafka-python.readthedocs.io/en/master/apidoc/KafkaConsumer.html
This way should be more flexible to alter between single and many servers, but I guess the flexible way won't work without some additional code :)